requirements.in

```diff
@@ -47,7 +46,7 @@ pytest-env==1.1.3
 python-decouple==3.8
 requests==2.31.0
 requests-mock==1.11.0
-ruff==0.1.3
+ruff==0.2.2
```

Comment:
While switching to ruff format, I updated ruff to the latest version which is 0.2.2.